scripts/skills/actives/rf_cheap_trick_skill.nut

```diff
@@ -0,0 +1,96 @@
+this.rf_cheap_trick_skill <- ::inherit("scripts/skills/skill", {
+	m = {
+		MeleeSkillModifier = 25,
+		RangedSkillModifier = 25,
+		DamageMult = 0.50,
+
+		// Private
+		WasUsedThisTurn = false,
```

Comment:
Minor point: in a lot of places we use `IsSpent` for this kind of flag, so for consistency you could use that name.